ludwig/automl/base_config.py

```diff
@@ -118,6 +127,33 @@ def _get_stratify_split_config(field_meta: FieldMetadata) -> dict:
     }
 
 
+def get_default_automl_hyperopt() -> Dict[str, Any]:
+    """Returns general, default settings for hyperopt.
+
+    For example:
+    - We set a random_state_seed for sample sequence repeatability
+    - We use an increased reduction_factor to get more pruning/exploration.
+
+    TODO: If settings seem reasonable, consider building this into the hyperopt schema, directly.
```

Comment:
I believe this is almost the default in the schema already, the only different being that the default scheduler is FIFO instead of async_hyperband, but I think async_hyperband is a good choice for automl config generation so we should stick to this for now.

Reply:
If async_hyperband is a clearly better option to use, then that's a good reason to make that the schema default. What are your thoughts on keeping FIFO the schema default?

Reply:
I think it's a safe change to make